### PR TITLE
Add tests from Diffblue Deeptest

### PR DESCRIPTION
**Pull request by paulkennethkent**

### CODE REVIEW

File: hystrix-core/src/test/java/com/netflix/hystrix/util/LongAdderTest.java [hunk 1]
```diff
@@ -0,0 +1,355 @@
+package com.netflix.hystrix.util;
```
Comment (sudo-jaa) [inline hystrix-core/src/test/java/com/netflix/hystrix/util/LongAdderTest.java]: LongAdder tests look good. Were there any tests generated that give a non-zero output?

File: hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+
+  /* testedClasses: LongMaxUpdater */
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers the entire method.
+   */
+
+  @Test
+  public void doubleValueOutputPositive() throws InvocationTargetException {
```
Comment (sudo-jaa) [inline hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java]: :+1:

File: hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+  }
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 95 branch to line 104
+   */
+
+  @Test
+  public void maxOutputZero000b7fdab3b5c2ffe3e() throws InvocationTargetException {
```
Comment (petersear) [inline hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java]: remove hash (or whole test?)

File: hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java [hunk 4]
```diff
@@ -0,0 +1,457 @@
...
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 95 branch to line 96
+   *
+   */
+
+  @Test
+  public void maxOutputZero00102ca13540c524591() throws InvocationTargetException {
```
Comment (petersear) [inline hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java]: remove hash (or whole test?)

File: hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java [hunk 5]
```diff
@@ -0,0 +1,457 @@
...
+  }
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 133 branch to line 145
+   */
+
+  @Test
+  public void maxThenResetOutputZero000d327998c16b695ed() throws InvocationTargetException {
```
Comment (petersear) [inline hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java]: Hash

File: hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java [hunk 6]
```diff
@@ -0,0 +1,457 @@
...
+
+  /*
+   * Test generated by Diffblue Deeptest.
+   * This test case covers:
+   *  - conditional line 133 branch to line 134
+   *
+   */
+
+  @Test
+  public void maxThenResetOutputZero001917efcd58cc6ae31() throws InvocationTargetException {
```
Comment (petersear) [inline hystrix-core/src/test/java/com/netflix/hystrix/util/LongMaxUpdaterTest.java]: Hash

**Review by petersear — CHANGES_REQUESTED** [review]

**Review by petersear — APPROVED** [review]

File: hystrix-core/src/test/java/com/netflix/hystrix/util/LongAdderTest.java (hunk 1 above)
Comment (paulkennethkent) [reply hystrix-core/src/test/java/com/netflix/hystrix/util/LongAdderTest.java]: unfortunately not.

**Review by JohnLBergqvist — APPROVED** [review]
All fine

**Review by sudo-jaa — APPROVED** [review]

**Pull request closed without merging**